### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)

### ISSUE THREAD

**Issue by CalcProgrammer1** [sn4k3/UVtools#826]
### System

```Text
UVtools v4.0.6 X64
Operative system: Linux 6.6.10-arch1-1 #1 SMP PREEMPT_DYNAMIC Fri, 05 Jan 2024 16:20:41 +0000 X64
Processor: AMD Ryzen 9 3950X 16-Core Processor
Processor cores: 32
Memory RAM: 23.69 / 31.26 GB
Runtime: arch-x64
Framework: .NET 6.0.25
AvaloniaUI: 11.0.6
OpenCV: 4.8.1

Path:       /tmp/.mount_UVtoolhPtnOa/usr/bin/
Executable: /tmp/.mount_UVtoolhPtnOa/usr/bin/UVtools
```

### Printer and Slicer

- Printer: 
- Slicer: 

### Description of the bug

```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Method Destroy is not implemented on interface org.freedesktop.IBus.Service
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: UVtools_demo_file.sl1  [Version: 0] [Class: SL1File]

Machine date time: 1/16/2024 11:48:45PM
    UTC date time: 1/17/2024 5:48:45AM
```

### How to reproduce

Open the tool, open a file.  It happened with the AUR version and with the AppImage version.

**Comment by CalcProgrammer1** [comment]
Duplicate of https://github.com/sn4k3/UVtools/issues/803

Closing.  Tried older version, it worked.
